### ISSUE TITLE
Allow configuring if a message without a message type is editable, resubmittable

### ISSUE THREAD

**Issue by alechenninger** [esbtools/esb-message-admin#164]
In #163 we defaulted these to `false` but IMO `null` should effectively be a valid entry among types of messages which are editable, etc.

**Comment by alechenninger** [comment]
Use case: my message was mostly good but for one reason or another consumer was not able to determine its message type so it is sent to EMA. I want to be able to edit the message, fix the issue that prevented consumer from being able to determine message type, and resubmit.

**Comment by macgregor** [comment]
Yeah, I honestly wasnt sure what to default it to. I dont entirely understand the reasoning behind allowing edit/resubmit for some messages and not other so i defaulted to the safe option. Easy enough to change.
